src/main/webapp/search-interview.html

```diff
@@ -23,7 +23,7 @@
         <button class="navbar-toggler" type="button" data-toggle="collapse" 
           data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" 
           aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
+        <span class="navbar-toggler-icon"></span>
```

Comment:
Shouldn't this one still be indented since it's inside the button tag?

Reply:
Yep, so hard to find a good formatter :/